torch/csrc/cudnn/Descriptors.h

```diff
@@ -62,8 +62,7 @@ struct ConvolutionDescriptor
   ~ConvolutionDescriptor() {
     cudnnDestroyConvolutionDescriptor(desc);
   }
-  void set(cudnnDataType_t dataType, int dim, int* pad, int* stride) {
-    int upscale[3] = {1, 1, 1};
+  void set(cudnnDataType_t dataType, int dim, int* pad, int* stride, int * upscale) {
```

Comment:
No particular reason. It was called upscale here before, and cudnn.h the corresponding parameter is called dilation, so I used dilation everywhere without thinking much about it.